web/rainmaker/dev-packages/egov-hrms-dev/src/ui-utils/commons.js

```diff
@@ -41,6 +42,7 @@ export const getSearchResults = async (queryObject, dispatch) => {
       "",
       queryObject
     );
+    response.Employees[0].user.photo= getphotoFileStoreId();
```

Comment:
By this way it will set same photo to all employee's that you update through hrms update